### DISCUSSION TITLE
405 Metod Not Allowed when redirected from authentication

### DISCUSSION THREAD

**Discussion by vedeja** [structurizr/onpremises#73] [Q&A] [answered]
### Question

We have SAML authentication set up with an Azure AD. It works great about half of the times. The other half we get 405 Metod Not Allowed when redirected from authentication, back to {structurizr.url}/saml/SSO. It starts working again after some time, some reloading or starting an incognito browser session. I’m not at all ruling out that the problem is in the AD, but would just like to see if someone here recognizes the problem.

**Comment by simonbrowndotje** [comment, marked as answer]
HTTP 405 ... this has been discussed a few times (https://github.com/search?q=repo%3Astructurizr%2Fonpremises+405&type=issues), and it's also mentioned in the docs -> https://docs.structurizr.com/onpremises/troubleshooting#saml-integration

**Reply by vedeja** [reply]
I'm a novice community member it seems :) I should have searched more. Please close this discussion.
